src/ivc/fold_relaxed_plonk_instance_chip.rs

```diff
@@ -1338,13 +1332,14 @@ mod tests {
                         as_bits: r,
                     };
 
-                    let m_bn = config.fixed_cycle_assigner().assign_all_fixed(
+                    config.fixed_cycle_assigner().assign_all_fixed(
                         &mut ctx,
                         || "m_bn",
                         scalar_module_as_limbs::<C1>(LIMB_WIDTH, LIMBS_COUNT)
                             .unwrap()
                             .into_iter(),
                     )?;
```

Comment:
You still don't need all this expression

Reply:
oh, missed this one.